confuse.py

```diff
@@ -778,19 +778,37 @@ def check_plain(self):
         plain = super(Loader, self).check_plain()
         return plain or self.peek() == '%'
 
+    @staticmethod
+    def override(loader):
```

Comment:
Let's maybe call this `add_constructors` or similar? I fear that "override" may sound a little too general.